### PR TITLE
Migrate .packages to package_config

### PR DESCRIPTION
**Pull request by annagrin**
Closes https://github.com/dart-lang/webdev/issues/1526

### CODE REVIEW

File: dwds/lib/src/readers/frontend_server_asset_reader.dart [hunk 1]
```diff
@@ -37,17 +37,17 @@ class FrontendServerAssetReader implements AssetReader {
...
     String outputPath,
     this._packageRoot,
   )   : _mapOriginal = File('$outputPath.map'),
         _mapIncremental = File('$outputPath.incremental.map'),
         _jsonOriginal = File('$outputPath.json'),
         _jsonIncremental = File('$outputPath.incremental.json'),
-        _packageConfig = loadPackageConfig(
-            File(p.absolute(p.join(_packageRoot, '.packages'))));
+        _packageConfig = loadPackageConfig(File(p
+            .absolute(p.join(_packageRoot, '.dart_tool/package_config.json'))));
```
Comment (annagrin) [inline dwds/lib/src/readers/frontend_server_asset_reader.dart:50]: '/' might not work on windows

Comment (annagrin) [reply dwds/lib/src/readers/frontend_server_asset_reader.dart:50]: nope it works:)

**Review by jakemac53 — APPROVED** [review]

File: frontend_server_common/lib/src/asset_server.dart [hunk 2]
```diff
@@ -12,7 +12,7 @@ import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:dwds/dwds.dart';
-import 'package:file/file.dart';
+import 'package:file/file.dart' hide Directory;
```
Comment (elliette) [inline frontend_server_common/lib/src/asset_server.dart:15]: I think hiding Directory can be removed now

Comment (annagrin) [reply frontend_server_common/lib/src/asset_server.dart]: Removed.

**Pull request merged**